URL formatting guide

burner1024 opened the pull request:
Added URL formatting guide, ref https://github.com/Gibberlings3/iesdp/pull/127#issuecomment-1099602205

lynxlynxlynx commented:
We have a contributing.md, so this and what snuck in before should be moved there instead. And just a link left in the README.

burner1024 commented:
OK I'll move the contents there.

> And just a link left in the README.

Sorry, didn't get this. Do you mean link Contributing in main README?

burner1024 commented:
Also @4Luke4 is [everything](https://github.com/burner1024/iesdp/blob/58354dc5b252e86ec4cd2065c2629d3a66d410bc/_data/README.md) clear? See any issues?

lynxlynxlynx commented:
Yeah, so people are less likely to miss those instructions.

burner1024 commented:
Moved to [contributing](https://github.com/burner1024/iesdp/blob/patch-2/CONTRIBUTING.md). Added relurl imports to corresponding files. Also added a minor css fix so that anchors that have code formatted text inside are still color blue.

lynxlynxlynx commented on file_formats/ie_formats/itm_v1.htm:
```diff
@@ -2,6 +2,7 @@
 title: "ITM file format"
 regenerate: true
 ---
+    {% include relurl.html %}
```
Aren't these changes noops? None of the links have been touched.

burner1024 replied:
They enable relurl to be work in the files. Without it, contributors following the guideline will find that it results in broken links.
I updated one link to check that it works.

lynxlynxlynx commented:
Thanks!
